AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/booking-details/booking-details.component.ts

```diff
@@ -223,7 +223,11 @@ export class BookingDetailsComponent implements OnInit, OnDestroy {
         if (status === UpdateBookingStatus.Cancelled) {
             this.showCancelBooking = false;
         }
+        this.hearing.Status = UpdateBookingStatus[status];
```

Comment:
Can this not just be:
`this.hearing.Status = status`
as status is already the correct type.

Reply:
I have now fixed to handle when status is failed so other unit test are not impacted.